packages/renderer/src/lib/container/ContainerDetailsLogsClear.svelte

```diff
@@ -0,0 +1,20 @@
+<script lang="ts">
+import { faEraser } from '@fortawesome/free-solid-svg-icons';
+import type { Terminal } from '@xterm/xterm';
+import Fa from 'svelte-fa';
+
+const { terminal }: { terminal: Terminal } = $props();
+
+function clear(): void {
+  terminal.clear();
+}
+</script>
+
+<div class="absolute top-0 right-0 px-1 z-50 m-1 opacity-50 space-x-1">
```

Comment:
The first Logs I went to I wouldn't have known the button was there if I wasn't looking. Maybe a little more opacity?
<img width="205" alt="Screenshot 2024-10-22 at 3 51 42 PM" src="https://github.com/user-attachments/assets/4aa26c9e-9911-4051-9a72-38f7e4caf46a">

Likewise, it is over top of the mem tooltip. z-order should be lowered so that tooltips remain on top.
<img width="205" alt="Screenshot 2024-10-22 at 3 53 41 PM" src="https://github.com/user-attachments/assets/49c7116a-293c-4984-9fe2-994b4132ce4e">